jwst/datamodels/schemas/keyword_msatargacq.schema.yaml

```diff
@@ -0,0 +1,120 @@
+%YAML 1.1
```

Comment:
"keyword_xxxx.schema" files are used to define individual keywords that then need to be inherited into other higher-level schemas, in particular for reference file datamodels that need only a few particular keywords to use as CRDS selectors. The data in this schema can be merged into the `msa_targ_acq.schema` below, such that there's a single schema that defines both header keywords and the table format for the MSA_TARG_ACQ extension.

Reply:
Moving all of the keyword*.schema information into the msatargacq.schema.yaml; which will contain all of the table definitions, and all (6+22) keyword definitions.